tests/integration/test_issue_2235.py

```diff
@@ -0,0 +1,54 @@
+import os
+import subprocess
+import sys
+
+import pytest
+
+from testing import run_pex_command
+
+
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="Flask 2.3.3 requires Python3.8+.")
+def test_standard_library_is_included(
+    tmpdir,  # type: Any
+):
+    # type: (...) -> None
+
+    built_pex_path = os.path.join(tmpdir, "flask.pex")
+    run_pex_command(
+        args=[
+            "Flask==2.3.3",
+            "-o",
+            built_pex_path,
+            "--venv",
+        ]
+    ).assert_success()
+
+    output_path = os.path.join(tmpdir, "output.txt")
```

Comment:
Does this work?! Perhaps it does in Python 3. As a general rule I do `os.path.join(str(tmpdir), "output.txt")` for tests, but perhaps that coercion is only needed for Python 2.7, which you don't test against here; so are safe?